src/views/ProfileEditView/index.js

```diff
@@ -0,0 +1,89 @@
+import React, { Component } from 'react';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import ProfileEditComponent from '../../components/Profile/ProfileEdit';
+import { profileUpdate, userImageUpload } from '../../redux/actions/userProfileActions/actions';
+
+class profileEditView extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      bio: '',
+      image: '',
+    };
+  }
+
+
+  componentWillReceiveProps(nextProps) {
```

Comment:
What is happening here?

Reply:
Hi,
The application state values are initially empty.
In order to update the state with new prop values this method 
_"componentWillReceiveProps(nextProps) "_ gets called whenever their is a change in prop values.

Regards
Steven Ennis